metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -200,6 +199,14 @@ class MetalsLanguageServer(
     MetalsLogger.setupLspLogger(workspace, redirectSystemOut)
     val clientExperimentalCapabilities =
       ClientExperimentalCapabilities.from(params.getCapabilities)
+
+    languageClient.underlying match {
+      case client: ConfiguredLanguageClient =>
+        client.configure(clientExperimentalCapabilities)
```

Comment:
Would it make sense to add this method to `MetalsLanguageClient` with a default implementation that ignores the argument?